pkg/controller/instanceset/object_builder.go

```diff
@@ -254,7 +254,7 @@ func injectCustomRoleProbeContainer(its *workloads.InstanceSet, template *corev1
 	agentPath := strings.Join([]string{roleAgentVolumeMountPath, roleAgentName}, "/")
 	initContainer := corev1.Container{
 		Name:            roleAgentInstallerName,
-		Image:           shell2httpImage,
+		Image:           strings.Join([]string{viper.GetString(constant.KBImageRegistry), shell2httpImage}, "/"),
```

Comment:
Should change shell2httpImage to `"apecloud/shell2http:1.16.0"`? Because the original repository `"msoap"` is not under our control. @free6om

Reply:
It's not like that, the whole path is like "docker.io/msoap/shell2http:1.16.0" or "apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com/msoap/shell2http:1.16.0".

Reply:
no any msoap group in apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com